ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-main.yaml

```diff
@@ -37,7 +37,7 @@ tests:
     export os=$(go env GOOS)
     export arch=$(go env GOARCH)
     export KB_TOOLS_ARCHIVE_NAME="kubebuilder-tools-${KUBEBUILDER_VERSION}-${os}-${arch}.tar.gz"
-    curl -s -f -L https://storage.googleapis.com/kubebuilder-tools/${KB_TOOLS_ARCHIVE_NAME}
+    curl -s -f -L https://storage.googleapis.com/kubebuilder-tools/${KB_TOOLS_ARCHIVE_NAME} -o ${KB_TOOLS_ARCHIVE_NAME}
     tar -zxvf ${KB_TOOLS_ARCHIVE_NAME}
     sudo mv -f ./kubebuilder /usr/local/kubebuilder
```

Comment:
this sudo usually prompt for a password. your best bet is just to use mv without sudo if this doesn't work

Reply:
Good point Mike. The other issue could be a read-only file system in which case I'm tempted to use kubebuilder from /tmp. Thanks!

Reply:
@fxiang1  may try to set the PATH to ./kubebuilder before running sonar cloud and reset PATH to original after sonar cloud scan.

Reply:
Thanks @xiangjingli I'll try to set the PATH and also KUBEBUILDER_ASSETS which overwrites the default kubebuilder path.